### ISSUE TITLE
Turn off videoCompression by default

### ISSUE THREAD

**Issue by jennifer-shehane** [cypress-io/cypress#26160]
### What would you like?

Cypress also automatically compresses the recorded video to a Constant Rate Factor (CRF) of 32. Cypress could turn off video compression by default for all users - having a `videoCompression`  value set to `false`  in their configuration file.

### Why is this needed?

Recording video in low CPU resourced machines results in slower runtime of Cypress tests. Video compression is on my default and blocking, so it increases the time the tests take to run. And video compression takes longer the longer the video is, so these reasons are compounding the problem. 

### Other

Turning off video compression would result in larger video sizes, which still needs exploration as to the impact for users and Cypress.

The release and implementation strategy has not been decided, but this could result in a breaking change for users.

**Comment by csvan** [comment]
Would be amazing if this could be revisited while on the topic of video: https://github.com/cypress-io/cypress/issues/21976

**Comment by jennifer-shehane** [comment]
We have decided to have `videoCompression` set to `false` for all users by default. This will go out in a breaking change (v13 likely).

In looking at this issue, we also fixed a bug where setting `videoCompression: 0` would result in broken videos [here](https://github.com/cypress-io/cypress/issues/5191).

We also intend to fix this issue where the video location doesn't print when video compression is skipped, since this situation would apply to all users now. [See issue](https://github.com/cypress-io/cypress/issues/25945)

We will also update the wording of what is printed to the terminal to say 'Compressing' instead of 'Processing' in the video terminal output. [See issue.](https://github.com/cypress-io/cypress/issues/26657)

We'll also add a 'true' value to videoCompression which coerces the value to 32 CRF. [See issue.](https://github.com/cypress-io/cypress/issues/26658)
